web/package.json

```diff
@@ -46,6 +47,7 @@
     "@angular/platform-browser-dynamic": "7.2.15",
     "@angular/platform-server": "7.2.15",
     "@angular/router": "7.2.15",
+    "@compodoc/compodoc": "^1.1.10",
```

Comment:
Should be a `devDependencies`, not need for the deployed version

Reply:
done